### ISSUE TITLE
Get and set anat/func resolution from data in fmriprep-options config

### ISSUE THREAD

**Issue by SucyLi** [FCP-INDI/C-PAC#1316]
Currently the resolution in fmriprep-options config is set based on HNU dataset and users have to change it manually when process a different dataset, which can be inconvenient and confusing. We can add an option called "native space", and then get and set data resolution within the pipeline.

**Comment by hahaai** [comment]
I think this should be a high priority. fmriprep-options is confusing with the hard-coded 3.4 resolutions in the config. Is that easy or hard to get the resolution from the native func?

**Comment by SucyLi** [comment]
> I think this should be a high priority. fmriprep-options is confusing with the hard-coded 3.4 resolutions in the config. Is that easy or hard to get the resolution from the native func?

I agree. It shouldn't be hard if using header information from nibabel. I can try to do that in fmriprep-options refactoring.
